dash/development/component_generator.py

```diff
@@ -83,10 +125,22 @@ def cli():
         default='package.json',
         help='The filename of the copied `package.json` to `project_shortname`'
     )
+    parser.add_argument(
+        '-r', '--rlang',
+        action='store_true',
+        help='Experimental: write DashR components to R dir, create R package'
+    )
+    parser.add_argument(
+        '--r-prefix',
```

Comment:
what if we made this required to make an R component, that way we would be able to get rid of the other arg, and we would remove the magic for `html` and `core` as well

Reply:
:hocho: :santa: